salt/modules/aptpkg.py

```diff
@@ -1226,6 +1226,9 @@ def upgrade(refresh=True, dist_upgrade=False, **kwargs):
 
         .. versionadded:: 2015.8.0
 
+    allow_downgrades
+        Allow apt to downgrade packages without a prompt.
```

Comment:
Can you add a `.. versionadded:: 3005` for this new arg?

Reply:
Added in [f517f10](https://github.com/saltstack/salt/pull/61469/commits/f517f100e57b4ebfd4adbd0a45d7bc8bd64d0e65)